internal/parser/jsdoc.go

```diff
@@ -285,6 +286,7 @@ loop:
 	}
 
 	if len(comments) > 0 {
+		comments[len(comments)-1] = strings.TrimRightFunc(comments[len(comments)-1], unicode.IsSpace)
```

Comment:
Ugh, so there was a place where this mattered? can we instead post-process those away rather than trimming just one side of the comments or something?